New GAP logo (#379)
gap-system/GapWWW: logo/index.md

---
title: The GAP Logo
layout: default
has_children: true
permalink: /logo/
---

# The GAP Logo

On this page you can learn a bit more about the GAP Logo: how it was selected,
what it signifies, and also where to download it for use in e.g. presentations
or on posters.

> We also plan to eventually offer some merchandise (stickers, T-shirts etc.)

Author: Maybe say something like:
Eventually, it will be possible to buy merchandise (stickers, t-shirts etc.) with the logo.

Reviewer: ```suggestion
Eventually, it will be possible to buy merchandise (stickers, t-shirts etc.) with the logo.
```

with the logo.

> ## GAP logo contest aftermath

Author: Change to 

GAP logo contest results
